tests/test_perf.py

```diff
@@ -34,14 +35,11 @@ def system_profiler(tmp_path: Path, perf_mode: str, insert_dso_name: bool) -> Sy
 
 
 def make_system_profiler(tmp_path: Path, perf_mode: str, insert_dso_name: bool) -> SystemProfiler:
+    profiler_state = ProfilerState(Event(), str(tmp_path), False, insert_dso_name, CPU_PROFILING_MODE)
```

Comment:
Code reuse please - you can add a function `make_profiler_state` in the tests, it will have all the default values as your fixture currently has.
The fixture will call the function with no changed arguments.
This test will call it with `insert_dso_name=insert_dso_name`.